osmhelper/osm_helper.py

```diff
@@ -69,25 +84,19 @@ def read_stops_from_file():
             return {}
 
     @staticmethod
-    def get_routes(route_type, bbox, refresh=False):
+    def get_routes(bbox, tags, refresh=False):
         if refresh:
             print "Start with fresh routes"
             routes = {}
         else:
             routes = OsmHelper.read_routes_from_file()
 
         # get the master routes first
-        route_masters = OsmHelper.get_route_masters(route_type, bbox, refresh)
+        route_masters = OsmHelper.get_route_masters(bbox, tags, refresh)
 
         api = overpy.Overpass()
-
-        result = api.query("""
-        <query type="relation">
-            <has-kv k="route" v="%s"/>
-            <bbox-query e="%s" n="%s" s="%s" w="%s"/>
-        </query>
-        <print/>
-        """ % (route_type, str(bbox["e"]), str(bbox["n"]), str(bbox["s"]), str(bbox["w"])))
+        query_str = 'relation(%s)%s;(._;);out body;' % (bbox, tags)
```

Comment:
I can't read and maintain these queries anymore, but I hope for comments when we optimize the queries ;)

How is `<has-kv k="route" v="%s"/>` represented in this new query?